Resources/doc/reference/configuration/headers.rst

```diff
@@ -218,7 +243,12 @@ This value must be a valid input to ``DateTime``.
                     headers:
                         last_modified: "-1 hour"
 
-.. hint::
+.. note::
+
+    Do **not** use last modified and the ETag together, as this only generates
+    confusion. In most situations, you should prefer the ETag method. Last
```

Comment:
hm, indeed. i still don't see the use, especially with what we offer (a hardcoded relative last-modified and an md5 hash on the content). maybe i just explain that unless you just want to provide a default last-modified because your logic will only put a last-modified in some situations, you should not activate this.

and on etag tell that if you have a custom strategy for last-modified, activating the md5 etag means that you still need to build the whole page to verify the etag. (well, technically the custom last modified listener could abort the request early with a 304 but it would not know if the etag matches, so there really is no point to it)

shall i do a sidebar that explains what each does and that having both means both must be validated according to the rfc?